internal/mode/static/nginx/runtime/manager.go

```diff
@@ -94,6 +117,36 @@ func (m *ManagerImpl) Reload(ctx context.Context, configVersion int) error {
 	return nil
 }
 
+// UpdateHTTPServers uses the NGINX Plus API to update HTTP upstream servers.
+// Only usable if running NGINX Plus.
+func (m *ManagerImpl) UpdateHTTPServers(upstream string, servers []ngxclient.UpstreamServer) error {
+	if !m.IsPlus() {
+		return errors.New("cannot update HTTP upstream servers: NGINX Plus not enabled")
+	}
+
+	added, deleted, updated, err := m.ngxPlusClient.UpdateHTTPServers(upstream, servers)
+	m.logger.V(1).Info(fmt.Sprintf("Added %d upstream servers", len(added)))
```

Comment:
shall we use keys rather than placing dynamic values inside the message? https://github.com/nginxinc/nginx-gateway-fabric/blob/main/docs/developer/logging-guidelines.md#message-guidelines